### DISCUSSION TITLE
In TE3 can't view the query used to import table into SSAS!

### DISCUSSION THREAD

**Discussion by TinyTimothy2** [TabularEditor/TabularEditor3#1233] [🙋♀️TE3 Q&A] [answered]
In TE3 can't view the query used to import table into SSAS
In Visual Studio i just click properties to view or edit it.
I can't find it anywhere in  a table in properties in TE3.
It is a show stopper.
Please help

**Comment by otykier** [comment, marked as answer]
Hi @TinyTimothy2 

Table import queries are located on the respective *partitions* within a table. Click on the partition to view the query in the **Expression Editor**.

In the screenshot below, there’s a single partition on the **Sales** table, containing an M query, but this could also have been a SQL query:

![image](https://github.com/TabularEditor/TabularEditor3/assets/8976200/c5665d40-5dcc-42bb-9732-486a37201587)
